test/jdk/java/awt/FullScreen/FullScreenInsets/FullScreenInsets.java

```diff
@@ -70,15 +78,16 @@ public static void main(final String[] args) {
             if (!device.isFullScreenSupported()) {
                 continue;
             }
+            System.out.println("Testing on Screen Device# "+ deviceCount++);
             device.setFullScreenWindow(wGreen);
             sleep();
-            testWindowBounds(device.getDisplayMode(), wGreen);
-            testColor(wGreen, Color.GREEN);
+            testWindowBounds(device.getFullScreenWindow().getBounds(), wGreen);
+            testColor(wGreen, Color.GREEN, "GREEN_" + deviceCount + ".png");
 
             device.setFullScreenWindow(wRed);
             sleep();
-            testWindowBounds(device.getDisplayMode(), wRed);
-            testColor(wRed, Color.RED);
+            testWindowBounds(device.getFullScreenWindow().getBounds(), wRed);
```

Comment:
You changed the code
from:
   comparing the bounds of the display mode and the bounds of the window
to:
   comparing the bounds of the window and the bounds of the window
I do not think it is the right thing to do.

Reply:
It was a miss - I meant to use the current ScreenDevice's DefaultConfiguration bounds - `device.getDefaultConfiguration().getBounds()` instead of `device.getFullScreenWindow().getBounds()`

I was debugging this further and found that the displayMode's width and height differ from the actual size of the full-screen windows as shown below for 2 screen extended display setup. The actual width and height of full screen window seem to match with `device.getDefaultConfiguration().getBounds()` and not with the display mode's which is what is being set here - https://github.com/openjdk/jdk/blob/2821fa9883cc9687b53e3bb5655732a614cff8fc/src/java.desktop/share/classes/java/awt/GraphicsDevice.java#L321

I'm currently looking into the past issues associated with this test case to understand the context of comparing the full screen window size to the size of display mode (if it was part of test requirement or issue to be solved).

(Tested on Windows)

```
No. of Screen Devices: 2

Testing on Screen Device# 0
Screen Device Default Config Bounds  Width: 1280.0  Height: 720.0
 Wrong window bounds: Expected: width = 1920, height = 1080 Actual: java.awt.Dimension[width=1280,height=720]
 Wrong window bounds: Expected: width = 1920, height = 1080 Actual: java.awt.Dimension[width=1280,height=720]
 
Testing on Screen Device# 1
Screen Device Default Config Bounds Width: 2293.0  Height: 960.0
 Wrong window bounds: Expected: width = 3440, height = 1440 Actual: java.awt.Dimension[width=2293,height=960]
 Wrong window bounds: Expected: width = 3440, height = 1440 Actual: java.awt.Dimension[width=2293,height=960]
```

Reply:
@mrserb I noticed that the difference in actual size vs display mode dimensions is because the actual size is scaled down by the device's current scale unlike display mode. This can be resolved by running the test at uiScale=1.